api/pkg/db/initializer/initializer.go

```diff
@@ -31,6 +32,13 @@ type Initializer struct {
 	api app.BaseConfig
 }
 
+var (
+	apiserverID       = 100000
+	apiserverName     = "api-server"
+	apiserverUserName = "apiserver"
+	apiserverEmailID  = "apiserver"
```

Comment:
probably we can append `-bot` in the suffix so that searching for a bot account is easy. WDYT?